### ISSUE TITLE
"Sorry, you are not allowed to access this page."

### ISSUE THREAD

**Issue by MadToban** [Bertholf/RB-Agency#11]
In the Agency settings, I'm getting this error when I click on TOOLS, SEARCH PROFILES, or MANAGE PROFILES. Sorry, can't find this issue/setting in the documentation.

**Comment by robertholf** [comment]
@MadToban I have seen this before but it was due to a missing record.  Do you have just the base or multiple plugins installed?  Glad to help track this down!

**Comment by MadToban** [comment]
I have the base and interact plugins installed.  
http://www.karicasting.ca/wp-admin/options-general.php?page=rb_agency_settings

**Comment by MadToban** [comment]
http://www.karicasting.ca/wp-admin/options-general.php?page=rb_agency_reports
http://www.karicasting.ca/wp-admin/options-general.php?page=rb_agency_profiles

These are also not responsive.

**Comment by robertholf** [comment]
@MadToban yes, the plugin was written before responsive was a thing... that old.. we refreshed the user facing screens but not the admin ones.  

RE: The Sorry not allowed, are you trying to access this URL directly or clicking from the menu?  Please advise steps and we will continue to investigate

**Comment by MadToban** [comment]
Clicking the tab in the menu.I'm not worried about responsiveness, full screen use is fine, just have to use it. :)Rod Salm                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the Rogers network.                                                                                                                                                                                                                From: Rob BertholfSent: Tuesday, May 15, 2018 9:55 AMTo: Bertholf/RB-AgencyReply To: Bertholf/RB-AgencyCc: MadToban; MentionSubject: Re: [Bertholf/RB-Agency] "Sorry, you are not allowed to access this page." (#11)@MadToban yes, the plugin was written before responsive was a thing... that old.. we refreshed the user facing screens but not the admin ones.
RE: The Sorry not allowed, are you trying to access this URL directly or clicking from the menu?  Please advise steps and we will continue to investigate

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/Bertholf/RB-Agency","title":"Bertholf/RB-Agency","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/Bertholf/RB-Agency"}},"updates":{"snippets":[{"icon":"PERSON","message":"@robertholf in #11: @MadToban yes, the plugin was written before responsive was a thing... that old.. we refreshed the user facing screens but not the admin ones.  \r\n\r\nRE: The Sorry not allowed, are you trying to access this URL directly or clicking from the menu?  Please advise steps and we will continue to investigate"}],"action":{"name":"View Issue","url":"https://github.com/Bertholf/RB-Agency/issues/11#issuecomment-389197197"}}}
{"@type":"MessageCard","@context":"http://schema.org/extensions","hideOriginalBody":"false","originator":"37567f93-e2a7-4e2a-ad37-a9160fc62647","title":"Re: [Bertholf/RB-Agency] \"Sorry, you are not allowed to access this page.\" (#11)","sections":[{"text":"","activityTitle":"**Rob Bertholf**","activityImage":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","activitySubtitle":"@robertholf","facts":[]}],"potentialAction":[{"name":"Add a comment","@type":"ActionCard","inputs":[{"isMultiLine":true,"@type":"TextInput","id":"IssueComment","isRequired":false}],"actions":[{"name":"Comment","@type":"HttpPOST","target":"https://api.github.com","body":"{\"commandName\":\"IssueComment\",\"repositoryFullName\":\"Bertholf/RB-Agency\",\"issueId\":11,\"IssueComment\":\"{{IssueComment.value}}\"}"}]},{"name":"Close issue","@type":"HttpPOST","target":"https://api.github.com","body":"{\"commandName\":\"IssueClose\",\"repositoryFullName\":\"Bertholf/RB-Agency\",\"issueId\":11}"},{"targets":[{"os":"default","uri":"https://github.com/Bertholf/RB-Agency/issues/11#issuecomment-389197197"}],"@type":"OpenUri","name":"View on GitHub"},{"name":"Unsubscribe","@type":"HttpPOST","target":"https://api.github.com","body":"{\"commandName\":\"MuteNotification\",\"threadId\":331890563}"}],"themeColor":"26292E"}

**Comment by clearlym-operations** [comment]
@MadToban Hi :) did some tests on our demo site. the URLs you provided are not correct.

.../wp-admin/options-general.php?page=rb_agency_profiles
should be: .../wp-admin/admin.php?page=rb_agency_profiles

.../wp-admin/options-general.php?page=rb_agency_reports
should be: .../wp-admin/admin.php?page=rb_agency_reports

**Comment by robertholf** [comment]
Ahh... I think it is the settings extension, we may want to disable that. so if you go to the plugin it shoudl work, but if you click on Settings it then brings you into options-general where it no longer has permission to go back to the plugin.

**Comment by MadToban** [comment]
Ok, how do I disable the settings extension?

**Comment by MadToban** [comment]
Ahh, I see how it works. Just go to the AGENCY tab, and use the search profiles there.

**Comment by robertholf** [comment]
Yes, we will add some logic checks, but that should handle it.  Let us know if there is anything else :)
